### PR TITLE
[Davranbek Jenisbaev]: Implement Comment Management (CRUD)

### PR DESCRIPTION
**Pull request by purple-mountain**
# Summary
- [x] Added comments controller, service, and repository for CRUD operations on comments.
- [x] Extended BlogsController and BlogsService to handle comments for blogs.
- [x] Added new migration for the Comment entity.
- [x] Added new tests for updated BlogsController.
- [x] Added new tests for updated BlogsService.
- [x] Added unit test for CommentsController.
- [x] Added unit test for CommentsService.
- [x] Added pagination and input validation for comments management.
- [x] Added env variable validation

### CODE REVIEW

File: src/modules/comments/comments.service.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { UpdateCommentRequestBodyDto } from "./dto/request/update-comment-request-body.dto";
+import { CommentsRepository } from "./comments.repository";
+import { NotFoundError } from "#/shared/errors/not-found-error";
+import { Comment } from "./entities/comment.entity";
+
+export class CommentsService {
```
Comment (mukhammadjon-ss) [inline src/modules/comments/comments.service.ts:9]: you have service methods for update and delete. Where is for creating and getting all methods ?

File: src/modules/comments/comments.controller.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import { checkResourceOwnership } from "#/shared/middlewares/checkResourceOwnership.middleware";
+import { validatePathParameter } from "#/shared/validators/path-parameter.validator";
+import { uuidSchema } from "#/shared/schemas/uuid-schema";
+import { auth } from "#/shared/middlewares/auth.middleware";
+import { validateRequestBody } from "#/shared/validators/request-body.validator";
+import { updateCommentRequestBodyDtoSchema } from "./dto/request/update-comment-request-body.dto";
+
+export const CommentsController = Router();
+
+CommentsController.put(
```
Comment (mukhammadjon-ss) [inline src/modules/comments/comments.controller.ts:55]: where are contollers for adding and retrieving comments

**Review by mukhammadjon-ss — CHANGES_REQUESTED** [review]

File: src/modules/comments/comments.controller.ts (hunk 2 above)
Comment (purple-mountain) [reply src/modules/comments/comments.controller.ts:55]: fixed. 
I moved them to comments controller from blogs controller

File: src/modules/comments/comments.service.ts (hunk 1 above)
Comment (purple-mountain) [reply src/modules/comments/comments.service.ts:9]: fixed. I moved methods for updating and deleting comments from blogs service to comments service

**Pull request merged**